O21.Game/CommandLineReporter.fs

```diff
@@ -0,0 +1,14 @@
+namespace O21.Game
+
+open System
+open O21.CommandLine
+
+type CommandLineReporter() =
+    interface IReporter with
+        member this.ReportInfo(text:string) = printfn $"%s{text}"
+        member this.ReportError(text:string) =
+            Console.ForegroundColor <- ConsoleColor.Red
+            printfn $"%s{text}"
+            Console.ForegroundColor <- ConsoleColor.White
+    member public this.ReportInfo(text:string) = (this :> IReporter).ReportInfo text
+    member public this.ReportError(text:string) = (this :> IReporter).ReportError text
```

Comment:
You don't need these methods.

F# interface implementations are always [explicit](https://learn.microsoft.com/en-us/dotnet/csharp/programming-guide/interfaces/explicit-interface-implementation) in terms of C#, this is why they sometimes feel a bit clunky.

But in this case, I can't see where these implementations are used or required.